flink-table/flink-table-planner/src/main/scala/org/apache/flink/table/planner/plan/utils/RelTreeWriterImpl.scala

```diff
@@ -188,5 +218,44 @@ class RelTreeWriterImpl(
         lastChildren = lastChildren.init
       }
     }
+    if (depth == 0 && statementNum == statementCnt && withAdvice) {
+      pw.println()
+      ADVICE_IDS.foreach(
+        advice =>
+          pw.println(s"advice[${advice._2}]: [${advice._1.getKind}] ${advice._1.getContent}"))
+      QUERY_LEVEL_ADVICE.forEach(
+        advice =>
+          pw.println(
+            s"advice[${NEXT_ADVICE_ID.getAndIncrement()}]: " +
+              s"[${advice.getKind}] ${advice.getContent}"))
+      if (ADVICE_IDS.isEmpty && QUERY_LEVEL_ADVICE.isEmpty) {
+        pw.println("No available advice...")
+      }
+    }
+  }
+
+  def resetDepth(): Unit = {
```

Comment:
the name `resetDepth` does not represent all behaviors here, how about naming it 'continue' ？and add some comments here